modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -4,30 +4,30 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-installing-aws-load-balancer-operator_{context}"]
-= Installing the AWS Load Balancer Operator
+= Installing AWS Load Balancer Operator by using the web console
 
-You can install the AWS Load Balancer Operator from the OperatorHub by using the {product-title} web console.
+You can install the AWS Load Balancer Operator by using the web console.
 
 .Prerequisites
 
 * You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
 
 * Your cluster is configured with AWS as the platform type and cloud provider.
 
+* If you are using a security token service (STS) or user-provisioned infrastructure, follow the related preparation steps. For example, if you are using AWS Security Token Service, see "Preparing for the AWS Load Balancer Operator on a cluster using the AWS Security Token Service (STS)".
+
 .Procedure
 
 . Navigate to *Operators* → *OperatorHub* in the {product-title} web console.
 . Select the *AWS Load Balancer Operator*. You can use the *Filter by keyword* text box or use the filter list to search for the AWS Load Balancer Operator from the list of Operators.
 . Select the `aws-load-balancer-operator` namespace.
-. Follow the instructions to prepare the Operator for installation.
-. On the *AWS Load Balancer Operator* page, click *Install*.
 . On the *Install Operator* page, select the following options:
 .. *Update the channel* as *stable-v1*.
 .. *Installation mode* as *A specific namespace on the cluster*.
 .. *Installed Namespace* as `aws-load-balancer-operator`. If the `aws-load-balancer-operator` namespace does not exist, it gets created during the Operator installation.
```

Comment:
Installation mode should be "All namespaces on the cluster (default)"

```
Installation mode
All namespaces on the cluster (default)
Operator will be available in all Namespaces.

A specific namespace on the cluster
This mode is not supported by this Operator
```

Reply:
ty